spec/models/additional_visitor_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe AdditionalVisitor do
+  it { should belong_to(:visit) }
```

Comment:
Same applies to the other tests. This is worth a read: http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/